### ISSUE TITLE
About IML-ViT's performance on Columbia dataset

### ISSUE THREAD

**Issue by clxhsa** [scu-zjz/IMDLBenCo#130]
### System Info

作者您好！我在初步尝试复现IML-ViT的结果时遇到了一些问题。我使用CASIAv2的5k张篡改图像训练200epoch后得到了如下结果：
CASIAv1: 0.6964
Coverage: 0.3925
Columbia: 0.5230
NIST16: 0.2741

<img width="702" height="180" alt="Image" src="https://github.com/user-attachments/assets/51180f17-e484-47cf-95c4-1b9410a08e16" />

对比您原文中给出的结果我发现差异主要出现在Columbia的表现上。对于Columbia的mask，我将原数据集中绿色的部分转为篡改区域

<img width="2286" height="1137" alt="Image" src="https://github.com/user-attachments/assets/555151fe-1139-4295-b5a4-d847eb2cc0c2" />

随后的预处理流程就是将所有数据集中size>1024的长边resize到1024，size<1024的样本保持不变，训练过程中进行正常的padding和augmentation。超参数使用您给出的最佳超参，gpu为四张5090。由于接触IML领域不久，所以不知道其他可能的问题出在哪里，希望能得到您的指导和帮助，谢谢！

### Reproduction

base_dir="./train_dir/iml-vit_mvss"
mkdir -p ${base_dir}

PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True \
CUDA_VISIBLE_DEVICES=4,5,6,7 \
torchrun  \
    --standalone    \
    --nnodes=1     \
    --nproc_per_node=4 \
./train.py \
    --model IML_ViT \
    --edge_lambda 20 \
    --vit_pretrain_path ./pretrain/iml-vit/mae_pretrain_vit_base.pth \
    --resume '' \
    --world_size 1 \
    --batch_size 2 \
    --data_path ./data_json/CASIAv2.json \
    --epochs 200 \
    --lr 1e-4 \
    --image_size 1024 \
    --if_padding \
    --min_lr 5e-7 \
    --weight_decay 0.05 \
    --edge_mask_width 7 \
    --test_data_path ./data_json/test.json \
    --warmup_epochs 2 \
    --output_dir ${base_dir}/ \
    --log_dir ${base_dir}/ \
    --accum_iter 8 \
    --seed 42 \
    --test_period 1 \
2> ${base_dir}/error.log 1>${base_dir}/logs.log

**Comment by SunnyHaze** [comment]
你好，感谢对我们工作的关注
目测超参没啥问题，可能就还是实验的时候有少量的随机性导致的ckpt问题，如果怀疑是数据集处理不一致问题，可以直接inference一下这个ckpt以这个为准。https://scu-zjz.github.io/IMDLBenCo-doc/zh/guide/quickstart/2_load_ckpt.html

**Comment by clxhsa** [comment]
好快的回复: )
用您提供的ckpt确实接近文中指标，我会再调整下训练参数试试，感谢！

**Comment by SunnyHaze** [comment]
如果数据集没啥问题，超参我觉得也基本是对的，我觉得可能就还是随机性相关的，比如换个seed train一把有时候就这样（同seed有时候现在深度学习有时候也固定不下来）。。。所以可能train几次大概对这个波动有个理解就好，不一定死磕。如果真要怀疑我可能还是怀疑的那个random crop augmentation（https://github.com/scu-zjz/IMDLBenCo/blob/9d150be192301b8c1f52d67cae4e6bab854fc3f2/IMDLBenCo/training_scripts/train.py#L156-L157 ）的概率和随机性可能有点波动。p调大点可能有点用？

**Comment by clxhsa** [comment]
okok我试下，非常感谢！👍👍
